app/models/user.rb

```diff
@@ -63,4 +65,12 @@ def badges
       Badge.find_by_badge_id(user_badge.badge_id).new
     }
   end
+
+  def completed_codewar_for_lesson(lesson)
+    codewars.each do |codewar|
+      return true if codewar.slug==lesson.codewars_challenge_slug
+    end
+    false
```

Comment:
added an issue for that, they just got the api working properly so will go ahead and add a column for language